server/src/main/java/org/apache/iotdb/db/rescon/SystemInfo.java

```diff
@@ -215,6 +246,10 @@ public synchronized void resetCompactionMemoryCost(long compactionMemoryCost) {
     this.compactionMemoryCost.addAndGet(-compactionMemoryCost);
   }
 
+  public synchronized void resetCompactionFileNumCost(int fileNum) {
```

Comment:
suggest to rename it to `decreaseCompactionFileNumCost`

Reply:
Resolved.